gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ClusterStats.java

```diff
@@ -15,6 +15,7 @@
 
 import com.google.common.collect.ImmutableMap;
 
+import java.io.Serializable;
```

Comment:
Please capitalize the commit message: https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages